components/data-oriented-programming/code/java-bbe/flexible-typing.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Be conservative in what you send, be liberal in what you accept'
-description: 'Ballerina employs "be conservative in what you send, be liberal in what you accept" by using structural types that support openness. <br><br>These types serve a dual purpose: enhancing static typing within programs and describing service interfaces accurately. While outgoing messages are tightly controlled to ensure protocol adherence, incoming data is handled with a degree of flexibility. The result is a balance of strictness and tolerance that enhances interoperability and resilience. This makes Ballerina a robust and adaptable choice for constructing cloud-native applications'
-image: 
+description: Ballerina employs be conservative in what you send, be liberal in what you accept by using structural types that support openness. <br><br>These types serve a dual purpose, enhancing static typing within programs and describing service interfaces accurately. While outgoing messages are tightly controlled to ensure protocol adherence, incoming data is handled with a degree of flexibility. The result is a balance of strictness and tolerance that enhances interoperability and resilience. This makes Ballerina a robust and adaptable choice for constructing cloud-native applications.
+image:
```

Comment:
If there is no image, you can remove the field from the frontmatter.

Reply:
Removed.